### PR TITLE
Update asc datapunt naar asc amsterdam MERGE FIRST

### PR DESCRIPTION
**Pull request by jasperswart**
- updated ASC-ui and asc-assets and removed asc-core 
- updated asc to version 0.26.1
- fixed issue with ASC Footer
- fixed issue with ASC SearchBar
- fixed issue with ASC Menu 
- please test this branch uitgebreid

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
   Link,
-  FooterHeading,
-} from '@datapunt/asc-ui';
+  themeColor,
+} from '@amsterdam/asc-ui';
 
 const StyledFooter = styled(AscFooter)`
-  .;
+  svg path {
+    fill: ${themeColor('tint', 'level1')} !important;
```
Comment (aditudorache) [inline src/components/Footer/index.tsx:18]: Is this `important` real necessary? It should be possible through a `&& > ` construction

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -1,22 +1,39 @@
...
+
+    :hover {
+      fill:  ${themeColor('tint', 'level1')} !important;
```
Comment (aditudorache) [inline src/components/Footer/index.tsx:21]: same here

File: src/components/Footer/index.tsx [hunk 3]
```diff
@@ -1,22 +1,39 @@
...
+
+  a {
+    color: ${themeColor('tint', 'level1')};
+
+    :hover {
+      color: ${themeColor('tint', 'level1')};
+      text-decoration: underline;
+
+      svg {
+        fill: green !important;
```
Comment (aditudorache) [inline src/components/Footer/index.tsx:33]: same here

**Review by aditudorache — CHANGES_REQUESTED** [review]
Nice work! I didn't test it visually though. Shouldn't this target `develop` branch? If we can remove the `importants` is production ready.

File: src/components/Footer/index.tsx (hunk 1 above)
Comment (jasperswart) [reply src/components/Footer/index.tsx:18]: unfortunately yes!

the footer should not have white color I think

File: src/components/Footer/index.tsx (hunk 2 above)
Comment (jasperswart) [reply src/components/Footer/index.tsx:21]: unfortunately yes!

File: src/components/Footer/index.tsx (hunk 3 above)
Comment (jasperswart) [reply src/components/Footer/index.tsx:33]: unfortunately yes!

**Review by aditudorache — APPROVED** [review]

**Pull request merged**